Sources/Networking/v1/APIRequestConfiguration.swift

```diff
@@ -24,23 +24,21 @@ extension APIRequest {
     public struct Configuration<QueryParams: Collection> where QueryParams.Element == (key: String, value: String) {
 
         let url: URL
-        let method: HTTPMethod
+        let method: APIRequest.HTTPMethod
         let queryParameters: QueryParams
         let allowedQueryReservedCharacters: CharacterSet?
         let headers: HTTPHeaders
         let body: Data?
         let timeoutInterval: TimeInterval
-        let attribution: URLRequestAttribution?
         let cachePolicy: URLRequest.CachePolicy?
 
         public init(url: URL,
-                    method: HTTPMethod = .get,
+                    method: APIRequest.HTTPMethod = .get,
                     queryParameters: QueryParams = [],
                     allowedQueryReservedCharacters: CharacterSet? = nil,
                     headers: APIRequest.Headers = APIRequest.Headers(),
                     body: Data? = nil,
                     timeoutInterval: TimeInterval = 60.0,
-                    attribution: URLRequestAttribution? = .developer,
```

Comment:
URLRequestAttribution was never used in any project and the apple default is .developer already, re-setting is redundant.